service/history/workflow/util.go

```diff
@@ -161,22 +164,21 @@ func TerminateWorkflow(
 // FindAutoResetPoint returns the auto reset point
 func FindAutoResetPoint(
 	timeSource clock.TimeSource,
-	badBinaries *namespacepb.BadBinaries,
+	ns VerifyBinaryChecksummer,
```

Comment:
It feel like taking a func here might be easier to read?
checksumVerifyFn func(string) error